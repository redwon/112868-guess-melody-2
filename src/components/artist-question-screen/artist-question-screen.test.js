import React from 'react';
import renderer from 'react-test-renderer';

import ArtistQuestionScreen from './artist-question-screen';

const mockQuestion = {
  type: `artist`,
  song: {
    artist: `artist`,
    src: `song`,
  },
  answers: [
    {
      picture: `pic`,
      artist: `artist`,
    },
  ],
};

it(`renders correctly`, () => {
  const tree = renderer
    .create(<ArtistQuestionScreen
      question={mockQuestion}
      screenIndex={0}
      onAnswer={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
