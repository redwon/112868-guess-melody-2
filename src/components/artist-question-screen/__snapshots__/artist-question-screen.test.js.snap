// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Кто исполняет эту песню?
  </h2>
  <div
    className="game__track"
  >
    <div
      className="track"
    >
      <button
        className="track__button track__button--play"
        onClick={[Function]}
        type="button"
      />
      <div
        className="track__status"
      >
        <audio
          src="song"
        />
      </div>
    </div>
  </div>
  <form
    className="game__artist"
  >
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-0"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="answer-0"
      />
      <label
        className="artist__name"
        htmlFor="answer-0"
      >
        <img
          alt="artist"
          className="artist__picture"
          src="pic"
        />
        artist
      </label>
    </div>
  </form>
</section>
`;
